tests/spdx3/bump/test_external_document_ref_bump.py

```diff
@@ -0,0 +1,36 @@
+# SPDX-FileCopyrightText: 2023 spdx contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+from spdx_tools.spdx3.bump_from_spdx2.checksum import bump_checksum
+from spdx_tools.spdx3.bump_from_spdx2.creation_information import bump_creation_information
+from spdx_tools.spdx3.bump_from_spdx2.external_document_ref import bump_external_document_ref
+from spdx_tools.spdx3.payload import Payload
+from spdx_tools.spdx.model import ExternalDocumentRef
+from tests.spdx.fixtures import checksum_fixture, creation_info_fixture, external_document_ref_fixture
+
+
+def test_bump_external_document_ref():
+    checksum = checksum_fixture()
+    external_document_ref = ExternalDocumentRef("DocumentRef-external", "https://external.uri", checksum)
+    namespace_map, imports = bump_external_document_ref(external_document_ref)
```

Comment:
```suggestion
    namespace_map, external_map = bump_external_document_ref(external_document_ref)
```

Reply:
Well, the property is actually called `imports`, with its type being `ExternalMap`, so I probably leave it like that for now.